pythran/pythonic/numpy/fft/irfft.hpp

```diff
@@ -43,24 +43,44 @@ namespace numpy
 
       // Create the twiddle factors. These must be kept from call to call as
       // it's very wasteful to recompute them.
+      // The call to npy_irfftf makes changes to the twiddle factor buffer (then
+      // restores it) so it's
+      // not thread safe. To avoid this:
+      // I keep one copy of the twiddle factors that no thread is allowed to
+      // modify. Then each thread
+      // has its own copy of it that it can modify at will.
+      // This is from fftpack_litemodule.c
+      static std::map<long, double *> all_twiddles_irfft_global;
```

Comment:
Can you do that with vector (and without vector copy of course)?

Reply:
Done. Let me know if that's what you had in mind. Indeed, it's a lot simpler that way!